LayoutTests/fast/dom/Range/ranges-across-trees.html

```diff
@@ -38,7 +38,7 @@
         shouldBeFalse("bodyRange().isPointInRange(shadowNode, 0)");
         shouldThrowErrorName("bodyRange().comparePoint(shadowNode, 0)", "WrongDocumentError");
         shouldBeFalse("bodyRange().intersectsNode(shadowNode)");
-        shouldThrowErrorName("bodyRange().compareNode(shadowNode)", "WrongDocumentError");
+        shouldThrowErrorName("bodyRange().compareNode(shadowNode)", "TypeError");
```

Comment:
I don't think it's helpful to keep this. We have the WPT test to be sure it's no longer implemented. We don't need a wrong document test for a function that does not exist.